src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -409,7 +409,7 @@ static int dev_acquisition_start(const struct sr_dev_inst *sdi)
 		return SR_ERR;
 
 	/* Reset all operational states. */
-	devc->rle_count = devc->sample_buf_size = 0;
+	devc->rle_count = 0;
```

Comment:
Probably correct, but maybe one sentence to the commit message explaining why it is now always up to date?

Reply:
Good catch! I amended the commit message (it's because the buffer and its size were reset in different functions, at different times).